packages/wonder-blocks-modal/components/modal-launcher/index.js

```diff
@@ -0,0 +1,140 @@
+// @flow
```

Comment:
hmm... I'm not a fan of the file naming of this component I think I'd prefer to just have all the js files be inside of the components directory. And if you really want this to be in a sub-directory then rename this file to be `modal-launcher.js` and just make the `index.js` file: `export * from "modal-launcher.js";`, like we've done elsewhere.

Reply:
Same.  I find it hard to differentiate files when there are multiple `index.js`s in a codebase.